lib/tests/delta_generator_test_case.py

```diff
@@ -44,8 +44,12 @@ def __init__(self):
 class DeltaGeneratorTestCase(unittest.TestCase):
     def setUp(self):
         self.forward_msg_queue = ForwardMsgQueue()
-        self.orig_report_ctx = None
-        self.new_script_run_ctx = ScriptRunContext(
+
+        # Save our thread's current ScriptRunContext
+        self.orig_report_ctx = get_script_run_ctx()
+
+        # Create a new ScriptRunContext to use for the test.
+        self.script_run_ctx = ScriptRunContext(
```

Comment:
This is just a name change from `DeltaGenerator.new_script_run_ctx` -> `DeltaGenerator.script_run_ctx`, for clarity.